newsletter/admin.py

```diff
@@ -43,13 +44,22 @@
 
 from .settings import newsletter_settings
 
+
+logger = logging.getLogger(__name__)
+
+
 # Contsruct URL's for icons
-ICON_URLS = {
-    'yes': '%snewsletter/admin/img/icon-yes.gif' % settings.STATIC_URL,
-    'wait': '%snewsletter/admin/img/waiting.gif' % settings.STATIC_URL,
-    'submit': '%snewsletter/admin/img/submitting.gif' % settings.STATIC_URL,
-    'no': '%snewsletter/admin/img/icon-no.gif' % settings.STATIC_URL
-}
+def icon_url(icon, alt_text, _urls={}):
+    if not _urls:
+        _urls['yes'] = static('newsletter/admin/img/icon-yes.gif')
+        _urls['wait'] = static('newsletter/admin/img/waiting.gif')
+        _urls['submit'] = static('newsletter/admin/img/submitting.gif')
+        _urls['no'] = static('newsletter/admin/img/icon-no.gif')
+
+    return format_html(
+        '<img src="{}" width="10" height="10" alt="{}"/>',
+        _urls[icon], alt_text
+    )
```

Comment:
I really do think this function deserves a (small) docstring explaining its usages.

Moreover, the `_urls` parameter is not used here, I prefer to not have it there for readability reasons.

Lastly, if you can, please make sure that this approach works well with a future migration to SVG icons - in line with Django's admin icons.